### PR TITLE
respect and pass null / omitted arguments

### PR DESCRIPTION
**Pull request by jchappelow**
Rebirth of https://github.com/kwilteam/kwil-db/pull/520

### CODE REVIEW

**Review by brennanjl — APPROVED** [review]
lgtm, aside from the one comment removing the ci change.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -143,6 +143,7 @@ jobs:
           rm -rf /tmp/kgw
           git clone https://github.com/kwilteam/kgw.git /tmp/kgw
           cd /tmp/kgw
+          git checkout action-call-workaround # REMOVE!
```
[the remark this answers is not in the document]
Comment (jchappelow) [reply .github/workflows/ci.yaml:146]: Oh, this means remove after this PR merges and then kgw is updated for the new payload.

**Review by Yaiba — APPROVED** [review]
I did review yesterday, not sure if you have checked it.
Just repost here https://github.com/kwilteam/kwil-db/pull/520#pullrequestreview-1901867206

**Comment by jchappelow** [comment]
Chatted with @Yaiba and we're removing the `Rest` field with the `rlp:"tail"` tags for now.  I think these are only sensible in communications payloads like perhaps `ActionCall`, but not `ActionExecution`, which is serialized into a block and is used to mutate state.

**Review by Yaiba — APPROVED** [review]
lgtm

**Pull request merged**